.gitignore

```diff
@@ -52,3 +52,6 @@ coverage/
 ## --- Hardhat Gitignore for Foundry artifacts ---
 artifacts/forge-std
 artifacts/ds-test
+
+## Ignore raw bytecode analyser cache
+tools/custom/raw-bytecode-analyser/bytecode_cache.sqlite
```

Comment:
I would move this part of `.gitignore` to a new `.gitignore` inside the `tools/custom/raw-bytecode-analyser/` folder so the folder becomes self-contained.